Sources/Apodini/Request/Action.swift

```diff
@@ -18,14 +18,17 @@ public enum Action<Element: Encodable>: ApodiniEncodable {
     /// as a final response to the client.
     /// Will be the last message on the stream sent by the server.
     case final(_ element: Element)
+    /// Closes the connection, without  sending a response.
+    case end
 
     func accept(_ visitor: ApodiniEncodableVisitor) {
         visitor.visit(action: self)
     }
 
     public func encode(to encoder: Encoder) throws {
         switch self {
-        case .nothing:
+        case .nothing,
+             .end:
             fatalError("Tried encoding Action.nothing!")
```

Comment:
A little nit here: In the case that we would reach the race condition that `.end` is encoded, the fatalError would still print "Tried encoding .nothing" which is a bit misleading I think, should we ever need to track down such an issue.
Maybe just make a extra case or put `\(self)` into the string.